### ISSUE TITLE
Problem reading a wav file (struct.error)

### ISSUE THREAD

**Issue by comiconomenclaturist** [devsnd/tinytag#64]
I have an issue with reading a couple of wav files. They were ripped from a CD using dBpoweramp, but TinyTag can't read them. The metadata can be read by ffmpeg though. 

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "tinytag/tinytag.py", line 131, in get
    tag.load(tags=tags, duration=duration, image=image)
  File "tinytag/tinytag.py", line 145, in load
    self._parse_tag(self._filehandler)
  File "tinytag/tinytag.py", line 860, in _parse_tag
    self._determine_duration(fh)  # parse whole file to determine tags:(
  File "tinytag/tinytag.py", line 851, in _determine_duration
    id3._parse_id3v2(fh)
  File "tinytag/tinytag.py", line 596, in _parse_id3v2
    frame_size = self._parse_frame(fh, id3version=major)
  File "tinytag/tinytag.py", line 629, in _parse_frame
    frame = struct.unpack(binformat, frame_header_data)
struct.error: unpack requires a string argument of length 10
```

**Comment by devsnd** [comment]
@comiconomenclaturist can you please provide a sample so i can reproduce the issue?

**Comment by comiconomenclaturist** [comment]
@devsnd here you go:

https://oto-public.s3-eu-west-1.amazonaws.com/02%20FME%20(Free%20Music%20Ensemble)%20-%20Part%202%20(for%20Paul%20Lytton).wav

**Comment by devsnd** [comment]
Hey @comiconomenclaturist 

Your issue has been fixed by @ianhomer in PR #67 

If you install the newest version from pypi you should be good to go!

**Comment by comiconomenclaturist** [comment]
great, thanks! I'll check it out
